eventserver/server_auth.go

```diff
@@ -9,12 +9,15 @@ import (
 
 var (
 	ErrInvalidCredentials = errors.New("invalid credentials")
-	ErrAdminOnly          = errors.New("billing data currently requires cloud_controller.admin or cloud_controller.global_auditor scope")
+	ErrAdminOnly          = errors.New("you need to be billing_manager or an administrator to retreive the billing data")
```

Comment:
I would rename the error variable to `ErrBillingPermissions` or something else. It is not for "Admins only" anymore

Reply:
Nice touch.